integration/test/ParseLiveQueryTest.js

```diff
@@ -135,4 +136,26 @@ describe('Parse LiveQuery', () => {
     await sleep(1000);
     assert.equal(count, 1);
   });
+
+  it('can subscribe to ACL', async () => {
+    const user = await Parse.User.signUp('ooo', 'password');
+    const ACL = new Parse.ACL(user);
+
+    const object = new TestObject();
+    object.setACL(ACL);
+    await object.save();
+
+    const query = new Parse.Query(TestObject);
+    query.equalTo('objectId', object.id);
+    const subscription = await query.subscribe(user.getSessionToken());
+    let count = 0;
+    subscription.on('update', object => {
+      count++;
+      assert.equal(object.get('foo'), 'bar');
+    })
+    await object.save({ foo: 'bar' });
+    await sleep(1000);
```

Comment:
I don't think we should introduce second long waits if we can avoid it.

you could use done in this case, and in the update handler, you can do all the asserts, logout and then use done.

Reply:
i know that this is just modeling other tests, but i don't like introducing another second wait